test/specs/scale.linear.tests.js

```diff
@@ -901,9 +901,9 @@ describe('Linear Scale', function() {
 		var yScale = chart.scales.y;
 		expect(xScale.paddingTop).toBeCloseToPixel(0);
 		expect(xScale.paddingBottom).toBeCloseToPixel(0);
-		expect(xScale.paddingLeft).toBeCloseToPixel(12);
-		expect(xScale.paddingRight).toBeCloseToPixel(13.5);
-		expect(xScale.width).toBeCloseToPixel(468 - 6); // minus lineSpace
+		expect(xScale.paddingLeft).toBeCloseToPixel(50);
```

Comment:
Looks like a big change here

Reply:
Yeah, did not figure that out yet. Might be the `_configure` moved earlier and offsets not being calculated, so tick label requires padding and so on.